pandas/core/generic.py

```diff
@@ -5906,6 +5906,95 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def as_nullable_dtypes(
```

Comment:
-1 on the name; something like: convert_dtypes, infer_dtypes i think is more appropriate.

Reply:
Resolving as discussion is below.